crates/provider-keyvalue-redis/src/lib.rs

```diff
@@ -276,7 +276,20 @@ impl keyvalue::batch::Handler<Option<Context>> for KvRedisProvider {
         keys: Vec<String>,
     ) -> anyhow::Result<Result<Vec<Option<(String, Bytes)>>>> {
         check_bucket_name(&bucket);
-        Ok(self.exec_cmd(ctx, &mut Cmd::mget(&keys)).await)
+        let data = match self
```

Comment:
why the change here?

Reply:
Because you have to massage the return data and map it to the Vec of tuples. So this fetches the data and then zips it up with the keys that were passed in